modules/bastion-group/main.tf

```diff
@@ -42,16 +42,30 @@ module "iap_bastion" {
 
 module "mig" {
   source  = "terraform-google-modules/vm/google//modules/mig"
-  version = "1.3.0"
+  version = "~> 3.0"
 
-  project_id  = var.project
-  region      = var.region
-  target_size = var.target_size
-  hostname    = var.name
-  hc_port     = 22
+  project_id        = var.project
+  region            = var.region
+  target_size       = var.target_size
+  hostname          = var.name
+  instance_template = module.iap_bastion.instance_template
 
-  tcp_healthcheck_enable = true
-  instance_template      = module.iap_bastion.instance_template
+  health_check = {
```

Comment:
Is there a reason why we don't want to parameterize this with a default val like here?
https://github.com/terraform-google-modules/terraform-google-vm/blob/4424b23aa5cc953f6206eeb1b3430102c276a565/examples/mig/full/variables.tf#L195

Reply:
Done. Changed to var.